examples/CMakeLists.txt

```diff
@@ -9,18 +9,26 @@ set(EXAMPLES
 set(EXAMPLES_OPENMP threadsafe1d1 threadsafe2d2f)
 set(EXAMPLES_C guru1d1c simple1d1c simple1d1cf)
 
+find_library(MATH_LIBRARY m)
+
 foreach(EXAMPLE ${EXAMPLES})
   add_executable(${EXAMPLE} ${EXAMPLE}.cpp)
   target_compile_features(${EXAMPLE} PRIVATE cxx_std_17)
```

Comment:
Maybe this is not needed either. Finufft public interface is c99 compatible. The example itself I think it uses c++11 only

Reply:
I tested building on Windows, Linux, and Mac (Clang, arm64). For the latter, I needed at least `cxx_std_14`.